modules/projects/projectList.php

```diff
@@ -8,7 +8,8 @@
     mysqli_query($db, $deleteProject);
 }
 
-function getProjects($db) {
+function getProjects($db,$isFiltered) {
+    if($isFiltered = false){
```

Comment:
not DRY

`if ($isFiltered) {
     $valueToSearch = $_POST["valueToSearch"];
     $countTotalProjectsQuery = "SELECT COUNT(*) FROM projects WHERE title LIKE '%$valueToSearch%'";
} else {
    $countTotalProjectsQuery = "SELECT COUNT(*) FROM projects";
}
`